### PR TITLE
Cs status testname comment

### PR DESCRIPTION
**Pull request by fischer-ncar**
Add test_name and comment to the output of cs_status

Test suite: Test with cesm2_0_beta01_gnu TestStatuses on yellowstone
Test baseline: 
Test namelist changes: 
Test status: 

Fixes: ESMCI/cime/#564

User interface changes?: 

Code review:

### CODE REVIEW

File: utils/python/CIME/test_status.py [hunk 1]
```diff
@@ -121,6 +121,15 @@ def get_status(self, phase):
     def get_comment(self, phase):
         return self._phase_statuses[phase][1] if phase in self._phase_statuses else None
 
+    def phase_statuses_dump(self):
+        if self._phase_statuses:
+                for phase, data in self._phase_statuses.iteritems():
```
Comment (jgfouca) [inline utils/python/CIME/test_status.py]: Some strange indenting here.

File: utils/python/CIME/test_status.py [hunk 2]
```diff
@@ -121,6 +121,15 @@ def get_status(self, phase):
...
+                    status, comments = data
+                    if not comments:
+                        print "%s %s %s" % (status, self._test_name, phase)
```
Comment (jgfouca) [inline utils/python/CIME/test_status.py]: We usually use logging.info to print stuff.

File: utils/python/CIME/test_status.py (hunk 1 above)
Comment (fischer-ncar) [inline utils/python/CIME/test_status.py]: Fixed the indenting.

File: utils/python/CIME/test_status.py (hunk 2 above)
Comment (fischer-ncar) [inline utils/python/CIME/test_status.py]: Switched to logging.info.

**Comment by jgfouca** [comment]
Thanks. I assume this PR does not need to go into maint branch.

**Pull request merged**